Backend/Remora.Discord.API/API/Formatting/Mention.cs

```diff
@@ -0,0 +1,61 @@
+//
+//  Mention.cs
+//
+//  Author:
+//       Jarl Gullberg <jarl.gullberg@gmail.com>
+//
+//  Copyright (c) 2017 Jarl Gullberg
+//
+//  This program is free software: you can redistribute it and/or modify
+//  it under the terms of the GNU Lesser General Public License as published by
+//  the Free Software Foundation, either version 3 of the License, or
+//  (at your option) any later version.
+//
+//  This program is distributed in the hope that it will be useful,
+//  but WITHOUT ANY WARRANTY; without even the implied warranty of
+//  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+//  GNU Lesser General Public License for more details.
+//
+//  You should have received a copy of the GNU Lesser General Public License
+//  along with this program.  If not, see <http://www.gnu.org/licenses/>.
+//
+
+using JetBrains.Annotations;
+using Remora.Discord.Core;
+
+namespace Remora.Discord.API.Formatting
+{
+    /// <summary>
+    /// Provides helper methods to mention various Discord objects.
+    /// </summary>
+    [PublicAPI]
+    public static class Mention
+    {
+        /// <summary>
+        /// Creates a mention string for a user.
+        /// </summary>
+        /// <param name="snowflake">The user Snowflake ID.</param>
+        /// <returns>
+        /// A user mention string.
+        /// </returns>
+        public static string User(Snowflake snowflake) => $"<@{snowflake.Value}>";
```

Comment:
There is also a member mention (it mentions the nickname specifically, where as user mentions the username even if they have a nickname) if I remember right, it's <@!id>

Reply:
There is, but it doesn't do anything different than the other one. Discord devs themselves recommend just using the straight mention.

Reply:
This only applies to mobile push notifications anyway afaik, not really worth implementing